### PR TITLE
P13b recovered gliders, P14 telemetered profiler moorings, P14a telemetered gliders

### PR DESCRIPTION
**Pull request by cdobs**
P13b recovered gliders:
-CP05MOAS-GL339 R00010
-CP05MOAS-GL376 R00011
-CP05MOAS-GL379 R00009
-CP05MOAS-GL514 R00001

P14 deployed profiler moorings:
-CP02PMCI D00014
-CP02PMCO D00014
-CP02PMUI D00015
-CP02PMUO D00015
-CP04OSPM D00013

P14a deployed gliders:
-CP05MOAS-GL339 D00011
-CP05MOAS-GL380 D00009
-CP05MOAS-GL388 D00010
-CP05MOAS-GL514 D00002
-CP05MOAS-PG583 D00002

### CODE REVIEW

File: CP05MOAS-GL380/CP05MOAS-GL380_D00009_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00009/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00009/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00009/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00009/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00009/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-05-PARADM000,telemetered,Available,
```
Comment (kuo-james) [inline CP05MOAS-GL380/CP05MOAS-GL380_D00009_ingest.csv:6]: Looks good

File: CP05MOAS-GL379/CP05MOAS-GL379_R00009_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00009/merged/cp_*.mrg,CP05MOAS-GL379-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00009/dvl/*.PD0,CP05MOAS-GL379-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00009/merged/cp_*.mrg,CP05MOAS-GL379-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00009/merged/cp_*.mrg,CP05MOAS-GL379-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00009/merged/cp_*.mrg,CP05MOAS-GL379-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00009/merged/cp_*.mrg,CP05MOAS-GL379-05-PARADM000,recovered_host,Available,
```
Comment (kuo-james) [inline CP05MOAS-GL379/CP05MOAS-GL379_R00009_ingest.csv:7]: Should the filename_mask be `/R00009/merged/` or `/R00009/merged-from-glider/` ?

File: CP02PMCI/CP02PMCI_D00014_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00014/imm/mmp/E*.DAT,CP02PMCI-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00014/imm/mmp/A*.DEC,CP02PMCI-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00014/imm/mmp/C*.DAT,CP02PMCI-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00014/imm/mmp/C*.DAT,CP02PMCI-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00014/imm/mmp/E*.DAT,CP02PMCI-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00014/imm/mmp/E*.DAT,CP02PMCI-WFP01-05-PARADK000,telemetered,Available,
```
Comment (kuo-james) [inline CP02PMCI/CP02PMCI_D00014_ingest.csv:10]: Looks good

File: CP02PMCO/CP02PMCO_D00014_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00014/imm/mmp/E*.DAT,CP02PMCO-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00014/imm/mmp/A*.DEC,CP02PMCO-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00014/imm/mmp/C*.DAT,CP02PMCO-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00014/imm/mmp/C*.DAT,CP02PMCO-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00014/imm/mmp/E*.DAT,CP02PMCO-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00014/imm/mmp/E*.DAT,CP02PMCO-WFP01-05-PARADK000,telemetered,Available,
```
Comment (kuo-james) [inline CP02PMCO/CP02PMCO_D00014_ingest.csv:10]: Looks good

File: CP02PMUO/CP02PMUO_D00015_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00015/imm/mmp/E*.DAT,CP02PMUO-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00015/imm/mmp/A*.DEC,CP02PMUO-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00015/imm/mmp/C*.DAT,CP02PMUO-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00015/imm/mmp/C*.DAT,CP02PMUO-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00015/imm/mmp/E*.DAT,CP02PMUO-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00015/imm/mmp/E*.DAT,CP02PMUO-WFP01-05-PARADK000,telemetered,Available,
```
Comment (kuo-james) [inline CP02PMUO/CP02PMUO_D00015_ingest.csv:10]: Looks good

File: CP02PMUI/CP02PMUI_D00015_ingest.csv [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMUI/D00015/imm/mmp/E*.DAT,CP02PMUI-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMUI/D00015/imm/mmp/A*.DEC,CP02PMUI-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMUI/D00015/imm/mmp/C*.DAT,CP02PMUI-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMUI/D00015/imm/mmp/C*.DAT,CP02PMUI-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMUI/D00015/imm/mmp/E*.DAT,CP02PMUI-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMUI/D00015/imm/mmp/E*.DAT,CP02PMUI-WFP01-05-PARADK000,telemetered,Available,
```
Comment (kuo-james) [inline CP02PMUI/CP02PMUI_D00015_ingest.csv:10]: Looks good

File: CP04OSPM/CP04OSPM_D00013_ingest.csv [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00013/imm/mmp/E*.DAT,CP04OSPM-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00013/imm/mmp/A*.DEC,CP04OSPM-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00013/imm/mmp/C*.DAT,CP04OSPM-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00013/imm/mmp/C*.DAT,CP04OSPM-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00013/imm/mmp/E*.DAT,CP04OSPM-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00013/imm/mmp/E*.DAT,CP04OSPM-WFP01-05-PARADK000,telemetered,Available,
```
Comment (kuo-james) [inline CP04OSPM/CP04OSPM_D00013_ingest.csv:10]: Looks good

File: CP05MOAS-GL339/CP05MOAS-GL339_D00011_ingest.csv [hunk 8]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-05-PARADM000,telemetered,Available,
```
Comment (kuo-james) [inline CP05MOAS-GL339/CP05MOAS-GL339_D00011_ingest.csv:6]: Looks good

File: CP05MOAS-GL339/CP05MOAS-GL339_R00010_ingest.csv [hunk 9]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00010/merged/cp_*.mrg,CP05MOAS-GL339-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00010/dvl/*.PD0,CP05MOAS-GL339-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00010/merged/cp_*.mrg,CP05MOAS-GL339-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00010/merged/cp_*.mrg,CP05MOAS-GL339-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00010/merged/cp_*.mrg,CP05MOAS-GL339-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00010/merged/cp_*.mrg,CP05MOAS-GL339-05-PARADM000,recovered_host,Available,
```
Comment (kuo-james) [inline CP05MOAS-GL339/CP05MOAS-GL339_R00010_ingest.csv:7]: Under filename_mask `/merged/cp_*.mrg` vs `/merged/cp_339*.mrg`

File: CP05MOAS-GL376/CP05MOAS-GL376_R00011_ingest.csv [hunk 10]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/R00011/merged/cp_*.mrg,CP05MOAS-GL376-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/R00011/dvl/*.PD0,CP05MOAS-GL376-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/R00011/merged/cp_*.mrg,CP05MOAS-GL376-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/R00011/merged/cp_*.mrg,CP05MOAS-GL376-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/R00011/merged/cp_*.mrg,CP05MOAS-GL376-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/R00011/merged/cp_*.mrg,CP05MOAS-GL376-05-PARADM000,recovered_host,Available,
```
Comment (kuo-james) [inline CP05MOAS-GL376/CP05MOAS-GL376_R00011_ingest.csv:7]: Looks good

File: CP05MOAS-GL379/CP05MOAS-GL379_R00009_ingest.csv (hunk 2 above)
Comment (cdobs) [reply CP05MOAS-GL379/CP05MOAS-GL379_R00009_ingest.csv:7]: For recovered data, /merged/ is correct
https://rawdata.oceanobservatories.org/files/CP05MOAS-GL379/R00009/

File: CP02PMCI/CP02PMCI_D00014_ingest.csv (hunk 3 above)
Comment (snwhite) [reply CP02PMCI/CP02PMCI_D00014_ingest.csv:10]: Looks good.

File: CP02PMCO/CP02PMCO_D00014_ingest.csv (hunk 4 above)
Comment (snwhite) [reply CP02PMCO/CP02PMCO_D00014_ingest.csv:10]: Looks good.

File: CP02PMUI/CP02PMUI_D00015_ingest.csv (hunk 6 above)
Comment (snwhite) [reply CP02PMUI/CP02PMUI_D00015_ingest.csv:10]: Looks good.

File: CP02PMUO/CP02PMUO_D00015_ingest.csv (hunk 5 above)
Comment (snwhite) [reply CP02PMUO/CP02PMUO_D00015_ingest.csv:10]: Looks good.

File: CP04OSPM/CP04OSPM_D00013_ingest.csv (hunk 7 above)
Comment (snwhite) [reply CP04OSPM/CP04OSPM_D00013_ingest.csv:10]: Looks good.

File: CP05MOAS-GL339/CP05MOAS-GL339_D00011_ingest.csv (hunk 8 above)
Comment (snwhite) [reply CP05MOAS-GL339/CP05MOAS-GL339_D00011_ingest.csv:6]: Looks good.

File: CP05MOAS-GL339/CP05MOAS-GL339_R00010_ingest.csv (hunk 9 above)
Comment (snwhite) [reply CP05MOAS-GL339/CP05MOAS-GL339_R00010_ingest.csv:7]: Looks good.

File: CP05MOAS-GL376/CP05MOAS-GL376_R00011_ingest.csv (hunk 10 above)
Comment (snwhite) [reply CP05MOAS-GL376/CP05MOAS-GL376_R00011_ingest.csv:7]: Looks good.

File: CP05MOAS-GL379/CP05MOAS-GL379_R00009_ingest.csv (hunk 2 above)
Comment (snwhite) [reply CP05MOAS-GL379/CP05MOAS-GL379_R00009_ingest.csv:7]: Looks good.

File: CP05MOAS-GL380/CP05MOAS-GL380_D00009_ingest.csv (hunk 1 above)
Comment (snwhite) [reply CP05MOAS-GL380/CP05MOAS-GL380_D00009_ingest.csv:6]: Looks good.

File: CP05MOAS-GL388/CP05MOAS-GL388_D00010_ingest.csv [hunk 11]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00010/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00010/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-05-PARADM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00010/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00010/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00010/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-03-CTDGVM000,telemetered,Available,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply CP05MOAS-GL388/CP05MOAS-GL388_D00010_ingest.csv:6]: Looks good.

File: CP05MOAS-GL514/CP05MOAS-GL514_D00002_ingest.csv [hunk 12]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-05-PARADM000,telemetered,Available,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply CP05MOAS-GL514/CP05MOAS-GL514_D00002_ingest.csv:6]: Looks good.

File: CP05MOAS-GL514/CP05MOAS-GL514_R00001_ingest.csv [hunk 13]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00001/merged/cp_*.mrg,CP05MOAS-GL514-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00001/dvl/*.PD0,CP05MOAS-GL514-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00001/merged/cp_*.mrg,CP05MOAS-GL514-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00001/merged/cp_*.mrg,CP05MOAS-GL514-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00001/merged/cp_*.mrg,CP05MOAS-GL514-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00001/merged/cp_*.mrg,CP05MOAS-GL514-05-PARADM000,recovered_host,Available,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply CP05MOAS-GL514/CP05MOAS-GL514_R00001_ingest.csv:7]: Looks good.

File: CP05MOAS-PG583/CP05MOAS-PG583_D00002_ingest.csv [hunk 14]
```diff
@@ -0,0 +1,8 @@
...
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-PG583-01-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-PG583-03-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-PG583-04-FLORTO000,telemetered,Available,
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-PG583-05-NUTNRM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-PG583-06-PARADM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-PG583-02-DOSTAM000,telemetered,Available,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply CP05MOAS-PG583/CP05MOAS-PG583_D00002_ingest.csv:8]: Looks good.

**Pull request merged**